src/app/components/Labled/LabeledInput.tsx

```diff
@@ -0,0 +1,32 @@
+import React from "react";
+import styles from "./LabeledInput.module.css";
+
+type LabeledInputProps = {
+  label: string;
+  placeholder: string;
+  value: string | number;
+  type?: string;
+  required?: boolean;
+  onChange: (value: string) => void;
+};
+
+export default function LabeledInput({
```

Comment:
What's the difference to `src/app/components/LabeledInput/LabeledInput`? Looks like a duplicate.